lib/utils/stackTraceParser.js

```diff
@@ -4,6 +4,8 @@ const escapeRegexp = require('escape-string-regexp');
 
 const isSuitestMethod = require('./isSuitestMethod');
 const {failedStackTrace} = require('../texts');
+const {config} = require('../../config');
+const logLevels = require('../constants/logLevels');
 
```

Comment:
If you decided to import config here, you can only check if logLevel is debug in stripSuitestCodeFromStack() instead of SuitestError and stackTraceWrapper

Reply:
Yes, I thought aboit it but I decide to leave helper functions pure.

Reply:
> Yes, I thought about it but I decide to leave helper functions pure.

I understand but stackTraceWrapper is also helper 8)